src/LoginFormContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { requestLogin, changeLoginFields, setAccessToken } from './actions';
+
+import { get } from './utils';
+
+import LoginForm from './LoginForm';
+
+export default function LoginFormContainer() {
+  const dispatch = useDispatch();
+
+  function handleSubmit() {
+    dispatch(requestLogin());
+  }
+
+  function handleChange({ name, value }) {
+    dispatch(changeLoginFields({ name, value }));
+  }
+
+  function handleClick() {
+    dispatch(setAccessToken(null));
+  }
+
+  const fields = useSelector(get('loginFields'));
+  const accessToken = useSelector(get('accessToken'));
+
+  if (accessToken) {
+    return (
+      <button
+        type="button"
+        onClick={handleClick}
+      >
+        Log out
+      </button>
+    );
+  }
+
+  return (
+    <LoginForm
+      fields={fields}
+      onSubmit={handleSubmit}
+      onChange={handleChange}
+    />
```

Comment:
```suggestion
    <>
      {accessToken ? (
        <LogoutForm onClick={handleClickLogout} />
      ) : (
        <LoginForm
          fields={loginFields}
          onChange={handleChange}
          onSubmit={handleSubmit}
        />
      )}
    </>
```

이렇게 하면 더 의미를 드러낼 수 있을 거 같아요.